src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -33,14 +33,21 @@
     public boolean failBuildIfStepFail;
     public CommandMode commandMode;
     public CLIRunner cliRunner;
+    private String logFolder;
+    private LogDestination logDestination = LogDestination.CONSOLE;
```

Comment:
These will be the defaults for a user that upgrades the plugin?

Reply:
Yes